ProcessMaker/Http/Resources/Process.php

```diff
@@ -14,14 +14,18 @@ public function toArray($request)
     {
         $array = parent::append('notifications', 'task_notifications')->toArray($request);
         $include = explode(',', $request->input('include', ''));
+        $filterWithoutAssignments = $request->input('filter_without_assignments', null);
         if (in_array('user', $include)) {
             $array['user'] = new Users($this->user);
         }
         if (in_array('category', $include)) {
             $array['category'] = new ProcessCategory($this->category);
         }
-        if (in_array('events', $include)) {
-            $array['events'] = $this->getStartEvents(true);
+        if (in_array('events', $include) && $filterWithoutAssignments) {
+            $array['events'] = $this->getStartEvents(true, true);
+        }
+        if (in_array('events', $include) && !$filterWithoutAssignments) {
+            $array['events'] = $this->getStartEvents(true, false);
```

Comment:
Could this be condensed to
```
$filterWithoutAssignments = $request->input('filter_without_assignments', false);
...
if (in_array('events', $include)) {
  $array['events'] = $this->getStartEvents(true, $filterWithoutAssignments);
}
```

Reply:
Thank you @nolanpro that was an elegant improvement!